src/Raven.Server/Documents/Handlers/CountersHandler.cs

```diff
@@ -213,6 +208,16 @@ public Task Get()
             return Task.CompletedTask;
         }
 
+        public static void GetInternal(DocumentDatabase database, DocumentsOperationContext context, StringValues counters, string docId, bool full, CountersDetail countersDetail)
```

Comment:
I'm wondering if there is a need to pass and fill the CounterDetails and not return them instead of void?

Reply:
fixed